README.md

```diff
@@ -95,32 +95,29 @@ If you are interested in making a tauri-app, please visit the [documentation web
 
 | Detail | Tauri | Electron |
 |--|--------|----------|
-| Binary Size MacOS | 0.6 MB (1) | 47.7 MB |
-| Memory Consumption MacOS | 13 MB | 34.1 MB |
+| Installer Size Linux | 3.1 MB | 52.1 MB |
+| Memory Consumption Linux | 180 MB | 462 MB |
+| Launch Time Linux | 0.39s | .80s |
 | Interface Service Provider | Varies | Chromium |
 | Backend Binding | Rust | Node.js (ECMAScript) |
 | Underlying Engine | C/C++ | V8 (C/C++) |
```

Comment:
```suggestion
| Underlying Engine | Now: C/C++ / Soon: Rust | V8 (C/C++) |
```

Reply:
We're going to have to change a lot of things in the readme when beta lands so I vote we just wait for that.